sonar-xml-plugin/char_before_prolog.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@vilchik-elena ha good point, thanks! Indeed this one is noise, the good one is under ``sonar-xml-plugin/src/test/resources/parsers/linecount/``